lua/loc/fr.json

```diff
@@ -19,51 +21,72 @@
   "bm_w_model70"        : "Winchester Modèle 70",
 
 	"bm_w_chinchilla"   : "Smith & Wesson Modèle 29",
-  
+
   "bm_w_baka"     : "Micro Uzi IMI",
 
   "bm_w_serbu"   : "Remington 870 à canon court",
-  
+
 	"bm_w_hunter" : "Amur Falcon Pistol Arbalète",
-	"bm_w_china"  : "China Lake Launcher",
+	"bm_w_china"  : "Lanceur China Lake",
 	"bm_w_ray"    : "M202 FLASH Lance-roquettes",
-    
+
 	"bm_wp_upg_o_eotech"        : "EOTech Modèle 553",
   "bm_wp_upg_o_t1micro"       : "Aimpoint  Micro T-1 monté sur rail",
 	"bm_wp_upg_o_docter"        : "Viseur DOCTER II Plus",
+	"bm_wp_upg_o_aimpoint"      : "Optique pour fusils Aimpoint PRO 12841",
 	"bm_wp_upg_o_eotech_xps"    : "EOTech Modèle EXPS3",
 	"bm_wp_upg_o_reflex"        : "Viseur Générique Open Reflex",
-	"bm_wp_upg_o_rx01"          : "Viseur Trijicon RX01 Polarized Reflex",
+	"bm_wp_upg_o_rx01"          : "Viseur Trijicon RX01 Reflex polarisé",
 	"bm_wp_upg_o_rx30"          : "Viseur Trijicon RX30 Reflex",
   "bm_wp_upg_o_ak_scopemount" : "Monture optique K-VAR KV-045",
-  "bm_wp_fps_upg_o_45iron"    : "Monture d'angle XTI Angle",	
+  "bm_wp_fps_upg_o_45iron"    : "Monture d'angle XTI",
   "bm_wpn_fps_upg_o_45rds_v2" : "Monture d'angle Aimpoint Micro T-2",
 	"bm_wp_upg_o_box"           : "Viseur Pulsar Digisight LRF N960 N970 nocturne avec chercheur de cibles",
   "bm_wpg_o_mbus_rear"        : "Mires avant et arrière MBUS",
   "bm_wp_upg_o_rmr"           : "Viseur Trijicon RM06 RMR",
   "bm_wp_upg_o_spot"          : "Double Monture Optique avancée NcSTAR",
 
+	"bm_wp_m4_m_straight" : "Magasin 22 cartouches STANAG",
```

Comment:
So, This one is a bit look, but writing the word "cartouche" (Rounds) shorter, is far from being common, Try with shortening Magasin first into Mag, and try to short it into "cart'" if you really need to optimise it.